### ISSUE TITLE
Real Steel

### ISSUE THREAD

**Issue by jerseysteal** [xenia-project/game-compatibility#424]
[Marketplace](http://marketplace.xbox.com/en-US/Product/REAL-STEEL/66acd000-77fe-1000-9115-d802584111e0)
Tested on (EmuCR-Xenia-20160322)

# Issues:
Game has a lot of slow down and glitched textures, from what i can tell the game works but isn't Ideal
*also audio lag

# Screenshot
http://imgur.com/l6lc4I3
# Labels:
gpu-missing-drawing slow-audio

**Comment by jerseysteal** [comment]
im new to this and would really like to help, can someone explain the release vs xenia i see it mentioned very often

**Comment by hckd01cn** [comment]
how to download

**Comment by shoopdahoop22** [comment]
with xbox live

**Comment by toro3555** [comment]
it doesnt even open the xex fille for me

**Comment by DragonSoul** [comment]
Game is quite playable, albeit there are a number of graphical issues and about a 2 second audio delay.  

Using latest build from the master branch: https://youtu.be/lveTDw-Py5U

**Comment by DragonSoul** [comment]
![real_steel 6-13-18 001](https://user-images.githubusercontent.com/40250959/41389649-d210eb3c-6f46-11e8-8944-93124b971623.PNG)

**Comment by DragonSoul** [comment]
state-gameplay?

**Comment by IPlayedAGameOnce** [comment]
As of late, the game works and looks much better then it did back in 2018. I've included pictures of the menu / "gameplay". I played through the entire game, and the only slow-downs that occurred was in the menus in the WRB, NOT in the Underworld. The game has no graphical issues that I could see when using D3D12, but under Vulkan, some textures have display issues (The timer, the standup count 1 - 10, etc). When the game has VSYNC disabled, the game runs 'faster', but it's not quite what you're thinking of (Still fun to mess with thou). 
![RS1](https://user-images.githubusercontent.com/73442832/97117408-fb042800-16d9-11eb-8de1-deaed063255f.PNG)
![RS2](https://user-images.githubusercontent.com/73442832/97117414-00fa0900-16da-11eb-83d5-b8aacea5f2a0.PNG)
![RS4](https://user-images.githubusercontent.com/73442832/97117417-08211700-16da-11eb-8263-b5d9942ba35c.PNG)
![RS6](https://user-images.githubusercontent.com/73442832/97117420-0c4d3480-16da-11eb-9e85-1ea34f5167f6.PNG)
![RS7](https://user-images.githubusercontent.com/73442832/97117424-0eaf8e80-16da-11eb-9252-6b52da9fb9fa.PNG)

**Comment by voltagex** [comment]
`marketplace-delisted`

**Comment by FelipeAlveSilva** [comment]
My xenia is closing after a few minutes of gameplay, what can it be?

**Comment by iforgetpasswordlol** [comment]
After 1-2 saves game crush but its not problem for me and in menu my minimum FPS is 30 but in game 61 without problems (well my pc very good)

**Comment by iforgetpasswordlol** [comment]
I still dont upgraded xenia + my drivers trash and now i have 10-15 fps or crash at menu

**Comment by iforgetpasswordlol** [comment]
Yeah... If i dont skip the game crash at intro so please add label intro

**Comment by spaki666** [comment]
how to download this game can you give me the link please

**Comment by spaki666** [comment]
![image](https://user-images.githubusercontent.com/108429506/176517252-7f28265e-c28c-4508-af6a-7efb70d24247.png)

what to do

**Comment by spaki666** [comment]
![image](https://user-images.githubusercontent.com/108429506/176707550-5b91a460-0881-484f-b2e9-3912b856537b.png)
help pls

**Comment by Margen67** [comment]
@spaki666 Stop spamming. Your comments were deleted because tech support belongs on the [Discord server](https://discord.gg/Q9mxZf9) like the README says.

**Comment by smv791** [comment]
how do i download it ?

**Comment by spaki666** [comment]
when will additional content and dlc be released?

**Comment by Caden2577** [comment]
I have trouble with it crashing after I press A to load into the game, anyone know how to fix it?

**Comment by soren66** [comment]
The game barely has any crashes on vulkan, But the glowing colors don't show up on vulkan

**Comment by FacuXD2024** [comment]
How do I download it?

**Comment by soren66** [comment]
In the pinned post at the Real steel subreddit.
r/realsteel

**Comment by ShadZ08** [comment]
can anyone help me download this game for free on pc + dlcs please?

**Comment by SturmWowa** [comment]
а как сделать светящиеся текстуры в игре, и в принципе нормальную графику?
